Runtime/DialogueViewBase.cs

```diff
@@ -204,34 +153,21 @@ public virtual void DialogueComplete()
         /// Signals that the user wants to go to the next line.
         /// </summary>
         /// <remarks>
-        /// <para>
         /// This method is generally called by a "continue" button, and
         /// causes the DialogueUI to signal the <see
         /// cref="DialogueRunner"/> to proceed to the next piece of
         /// content.
-        /// </para>
-        /// <para>
+        ///
         /// If this method is called before the line has finished appearing
-        /// (that is, before the status changes to 
-        /// <see cref="DialogueLineStatus.Finished"/>), the 
-        /// <see cref="DialogueRunner"/> will call <see cref="FinishRunningCurrentLine"/> 
-        /// on all <see cref="DialogueViewBase"/> derived classes and wait for them to finish
-        /// before calling <see cref="EndCurrentLine(Action)"/> on all of them.
-        /// </para>
+        /// (that is, before the line's status changes to 
+        /// <see cref="LineStatus.Delivered"/>), the line's status will
+        /// change to <see cref="LineStatus.Interrupted"/>, and <see
+        /// cref="OnLineStatusChanged"/> will be called to notify the view.
         /// </remarks>
         public void MarkLineComplete()
         {
-            if (dialogueRunnerCurrentLine) {
-                dialogueRunnerCurrentLine.OnViewUserIntentNextLine();
-            }
-        }
-
-        /// <summary>
-        /// Signals that the user wants to go to the end of the current line.
-        /// </summary>
-        public void FinishLine() {
-            if (dialogueRunnerCurrentLine) {
-                dialogueRunnerCurrentLine.OnViewUserIntentFinishLine();
+            if (controllingDialogueRunner) {
+                controllingDialogueRunner.OnViewUserIntentNextLine();
```

Comment:
I think this is a great improvement to how it was done before. But what if we pass the OnViewUserIntentNextLine() as Action from the DialogueRunner to the views via the `RunLine()` Method instead of letting the DialogueRunner set the reference to it's instance on an internal variable on all view class instances? That could make it even clearer and would be more in line with the way the Runner communicates with the view classes.

Reply:
Yeah, this is a good idea. Implemented.